public/plgapp/plgapp.js

```diff
@@ -94,6 +98,37 @@ var PlgApp = function () {
         }
         return new AppError(status + ' ' + xhr.status + ' ' + error, data);
     };
+
+    var setSessionTimeout = function(timeout_callback, seconds_before_timeout) {
+        if(!seconds_before_timeout) {
+            seconds_before_timeout = 0;
+        }
+        var session_expiry = getCookie('session_expiry') * 1000;
+        var now = (new Date()).valueOf();
+        var timeout = session_expiry - now - seconds_before_timeout * 1000;
```

Comment:
Identifier 'session_expiry' is not in camel case.<br>Identifier 'seconds_before_timeout' is not in camel case.